volumes/frus1865p4.xml

```diff
@@ -22919,8 +22919,7 @@
                     <opener>
                         <dateline rendition="#right">[<placeName>Bradford</placeName>, <date
                                 when="1865-04-28" ana="#date_undated-inferred-from-document-content"
-                                calendar="gregorian">April 28, 1865</date>
-                        </dateline>]</opener>
+                                calendar="gregorian">April 28, 1865</date>]</dateline>]</opener>
```

Comment:
Please check the brackets here (line 22922 of frus1865p4).

Reply:
See: https://github.com/HistoryAtState/frus/commit/4da0229aa7497ac243c2c863dceca0a8a5342039